services/app-api/src/resolvers/questionResponse/createQuestion.ts

```diff
@@ -77,6 +83,47 @@ export function createQuestionResolver(
             throw new UserInputError(errMessage)
         }
 
+        const conversionResult = convertContractWithRatesToFormData(
```

Comment:
Can we move away from converting the contract to the old HealthPlanPackage and HealthPlanFormData types? The rates refactor was the first step in deprecating those types. We're still converting to them in order to return them from our handlers, but when we convert the front end to be able to support rates across submissions the plan is to try and root those types out all together. 

I think the only real change this would require would be for the sendQuestionsStateEmail function to take in a ContractWithRatesType instead of a HPFD type, but it should have all the data you need to compose the email in it, then.